src/components/aboutUs/AboutUs.js

```diff
@@ -0,0 +1,70 @@
+import React, { useRef, useState, useEffect } from "react"
+import { Form, Button, Card } from "react-bootstrap"
+import { useHistory } from "react-router-dom"
+import "react-datepicker/dist/react-datepicker.css";
+import { useAuth } from "../../contexts/AuthContext"
+import Typography from "@material-ui/core/Typography";
+import { ExternalLinkIcon } from '@chakra-ui/icons'
+
+import {
+    Alert,
+    AlertIcon,
+    AlertTitle,
+    AlertDescription,
+    Box,
+    Input,
+    FormControl,
+    FormLabel,
+    FormErrorMessage,
+    FormHelperText,
+    Select,
+    Text,
+    Image,
+    Link
+} from '@chakra-ui/react';
+
+import NavigationBar from '../common/NavigationBar';
+
+import GOOSE_IMAGE from "../../images/Goose.png"
+
+import "../../styles/form-style.css";
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const CreateResource = () => {
+
+    return (
+        <div>
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 100, left: '50%', transform: 'translateX(-50%)' }}>
+                <Card style={{ padding: '16px' }}>
+                    <Text noOfLines={1} align="center" fontSize='4xl'>We are HonkedIn.</Text>
+                    <Image
+                        src={GOOSE_IMAGE}
+                        mx="auto"
+                        boxSize="200px"
+                        alt="Goose Image"
+                    />
+                    <Text align="center" fontSize='2xl' as='i'>Made by students, for students.</Text>
+                    <Text align="center" fontSize='2xl'>We aim to connect like-minded scholars at University of Waterloo.</Text>
+                </Card>
+                <br></br>
+                <Card style={{ padding: '16px' }}>
+                    <Text align="center" fontSize='2xl'>Want to learn more about how HonkedIn was built?</Text>
+                    <Link href='https://github.com/Safewaan/HonkedIn' isExternal align="center" target="_blank">
+                        Find out more here!<ExternalLinkIcon mx='2px' />
```

Comment:
the link icons are a nice addition but they aren't inline with the text

you don't have to fix this, I'm planning to come back to this page for styling (font, fontsize, margins, etc) anyways

Reply:
ok i can keep it for now